container-builder/util/log/log.go

```diff
@@ -16,116 +16,10 @@
 
 package log
 
-import (
-	"fmt"
-
-	"k8s.io/klog/v2"
-
-	"github.com/kiegroup/kogito-serverless-operator/container-builder/api"
+// Constants for different log level verbosity.
+const (
+	D = 3
+	I = 2
+	W = 1
+	E = 0
 )
```

Comment:
```suggestion
// Constants for different log level verbosity.
const (
        E = iota
        W
        I
        D
)
```

What about this? https://yourbasic.org/golang/iota/